### PR TITLE
breaching missiles breach windows and rwindows

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl:
fix: breaching missile works on windows with low bomb resistance
/:cl:

it didnt make sense that the breaching missile can break rwalls but not windows, windows with high bomb resist like plasma ones wont break

### CODE REVIEW

**Comment by Tlaltecuhtli** [comment]
hello bot is rogue?

File: code/modules/projectiles/projectile/special/rocket.dm [hunk 1]
```diff
@@ -83,7 +83,9 @@
 	/turf/closed,
 	/obj/vehicle/sealed/mecha,
 	/obj/machinery/door/,
```
Comment (Fikou) [inline code/modules/projectiles/projectile/special/rocket.dm:85]:
```suggestion
	/obj/machinery/door,
```

File: code/modules/projectiles/projectile/special/rocket.dm [hunk 2]
```diff
@@ -83,7 +83,9 @@
...
-	/obj/machinery/door/poddoor/shutters
+	/obj/machinery/door/poddoor/shutters,
+	/obj/structure/window/,
```
Comment (Fikou) [inline code/modules/projectiles/projectile/special/rocket.dm:87]:
```suggestion
	/obj/structure/window,
```

Comment (Fikou) [reply code/modules/projectiles/projectile/special/rocket.dm:87]: commit this whne bot is fixed to make bot green

Comment (Fikou) [reply code/modules/projectiles/projectile/special/rocket.dm:87]: bot fixesd??

Comment (Tlaltecuhtli) [reply code/modules/projectiles/projectile/special/rocket.dm:87]: doesnt that prevent subtypes from being selected

**Pull request merged**